src/material-experimental/mdc-form-field/_form-field-theme.scss

```diff
@@ -67,3 +72,27 @@
     }
   }
 }
+
+@mixin mat-mdc-form-field-density($config-or-theme) {
+  $density-scale: mat-get-density-config($config-or-theme);
+  @include _mat-form-field-density($density-scale);
```

Comment:
This is calling our legacy form field's density theme mixin, not the material-experimental one.

Should be changed to `@include _mat-mdc-form-field-density($density-scale);`